### PR TITLE
`treehouse services bookstack` (fixes #1340)

### LINKED ISSUE #1340: add service bookstack
https://hub.docker.com/r/linuxserver/bookstack

### PR DESCRIPTION
**Pull request by darthnoward**
fixes #1340 
```
root@treehouses:/home/pi/cli# ./cli.sh services bookstack install
Pulling bookstack_db ... done
Pulling bookstack    ... done
bookstack installed
modify default environment variables by running 'cli.sh services bookstack config edit'
root@treehouses:/home/pi/cli# ./cli.sh services bookstack up
valid yml
Creating network "bookstack_default" with the default driver
Creating bookstack_db ... done
Creating bookstack    ... done
bookstack built and started
root@treehouses:/home/pi/cli# ./cli.sh services bookstack start
Starting bookstack_db ... done
Starting bookstack    ... done
bookstack started
root@treehouses:/home/pi/cli# ./cli.sh services bookstack ps
1a66794a1d34        linuxserver/bookstack             "/init"                  31 seconds ago      Up 27 seconds            443/tcp, 0.0.0.0:6875->80/tcp                bookstack
f39af61e18c4        linuxserver/mariadb               "/init"                  33 seconds ago      Up 31 seconds            3306/tcp
```
successfully running on my device
HIMP - 2020-06-03 19:25:49

### CODE REVIEW

File: services/install-bookstack.sh [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    volumes:
+      - /srv/bookstack:/root/.bookstack
+    ports:
+      - 6875:80
+    restart: unless-stopped
+    depends_on:
+      - bookstack_db
+  bookstack_db:
+    image: linuxserver/mariadb
+    container_name: bookstack_db
```
Comment (law-lin) [inline services/install-bookstack.sh:30]: Delete the container_name field

Comment (law-lin) [reply services/install-bookstack.sh:30]: Same here

File: services/install-bookstack.sh [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+  # create service directory
+  mkdir -p /srv/bookstack
+
+  # create yml(s)
+  cat << EOF > /srv/bookstack/bookstack.yml
+version: "2"
+services:
+  bookstack:
+    image: linuxserver/bookstack
+    container_name: bookstack
```
Comment (law-lin) [inline services/install-bookstack.sh:13]: Same here as well

Comment (law-lin) [inline services/install-bookstack.sh:13]: Same here as well

**Comment by jayala-29** [comment]
I'm having the same issue @law-lin pointed out above

**Comment by jayala-29** [comment]
this looks good for me though
```
root@treehouses:~/cli# bats tests/services/bookstack.bats 
 ✓  services bookstack info
 ✓  services bookstack install
 ✓  services bookstack up
 ✓  services bookstack start
 ✓  services bookstack restart
 ✓  services available
 ✓  services running
 ✓  services running full
 ✓  services installed
 ✓  services installed full
 ✓  services ports
 ✓  services bookstack port
 ✓  services bookstack ps
 ✓  services bookstack url
 ✓  services bookstack autorun
 ✓  services bookstack autorun true
 ✓  services bookstack stop
 ✓  services bookstack down
 ✓  services bookstack icon
 ✓  services bookstack cleanup

20 tests, 0 failures
```

File: services/install-bookstack.sh (hunk 1 above)
Comment (darthnoward) [reply services/install-bookstack.sh:30]: Wonder if you can get it working with these changes?

File: services/install-bookstack.sh [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+EOF
+
+  # add autorun
+  cat << EOF > /srv/bookstack/autorun
+bookstack_autorun=true
+
+if [ "$bookstack_autorun" = true ]; then
+  treehouses services bookstack up
+fi
+
```
Comment (kjong) [inline services/install-bookstack.sh:60]: If you could add an additional space here (for formatting issues) that would be great.

Comment (kjong) [reply services/install-bookstack.sh:60]: Other than that, I got it up in about 2 minutes.

**Pull request merged**